utils/fake_libc_include/_fake_typedefs.h

```diff
@@ -150,8 +150,6 @@ typedef int uintmax_t;
 /* C99 stdbool.h bool type. _Bool is built-in in C99 */
 typedef _Bool bool;
 
-typedef int va_list;
```

Comment:
Sorry, I think it'd be actually better to keep this in this file and remove from the other. It's a typedef, after all... and it's needed to parse types, whereas fake_typedefs is needed for the preprocessor

Reply:
No problem. Like I said, I have no preference.